examples/testbed/2d.rs

```diff
@@ -28,21 +37,44 @@ enum Scene {
     Sprite,
 }
 
+#[cfg(feature = "bevy_ci_testing")]
+fn switch_scene_in_ci(
+    mut ci_config: ResMut<CiTestingConfig>,
+    scene: Res<State<Scene>>,
+    mut scenes_visited: Local<HashSet<Scene>>,
+    mut keyboard: ResMut<ButtonInput<KeyCode>>,
+    frame_count: Res<FrameCount>,
+    captured: RemovedComponents<Captured>,
+) {
+    if scene.is_changed() {
+        // Changed scene! trigger a screenshot in 100 frames, and reset keyboard state
+        ci_config.events.push(CiTestingEventOnFrame(
+            frame_count.0 + 100,
+            CiTestingEvent::NamedScreenshot(format!("{:?}", scene.get())),
+        ));
+        keyboard.release(KeyCode::Space);
```

Comment:
This business with `ButtonInput<KeyCode>` seems a bit janky.

Maybe we could we throw a `next` method on `scene`, and just manipulate `NextState<Scene>` here?